### ISSUE TITLE
Rewarded Videos not working

### ISSUE THREAD

**Issue by rajesh-mobicules** [floatinghotpot/cordova-admob-pro#420]
Hi,

Can I get a working code of rewarded videos with Mediation. I put the code which is present in the tutorials but its not working. Can you please suggest on this ?

**Comment by rajesh-mobicules** [comment]
Also I have to make it work with unity ads and Vungal ads in cordova based app. can you please suggest how to make it working ?

**Comment by floatinghotpot** [comment]
1. Add mediation library for unity ads and Vungle ads.
2. Configure mediation info in your AdMob portal.

**Comment by six60** [comment]
Hi @floatinghotpot 
I've tried adding the mediation library for unity ads but still can't get rewarded videos to play. 
This is the last issue before I can publish my app. I've looked through your Wiki/docs and can't find anything for this. I've looked at related issues and tried what was mentioned in them but am unable to get it working.

How do I properly add a mediation library to my app? (I'm using unity ads)
